formatters/symbol_formatter.js

```diff
@@ -30,20 +30,33 @@ class SymbolFormatter {
    *
    * @param {Result} result The result to format, must be valid
    * @param {string} ruleName The name of the rule this result is from
+   * @param {string} rulePolicyUrl The policyUrl of the rule this result is from
+   * @param {string} rulePolicyInfo The policyInfo of the rule this result is from
    * @param {string} errorSymbol The symbol to use if the result did not pass
    * @param {string} okSymbol The symbol to use if the result passed
    * @returns {string} The formatted string
    */
   static formatResult(
     result,
     ruleName,
+    rulePolicyUrl = undefined,
+    rulePolicyInfo = undefined,
     errorSymbol,
     okSymbol = logSymbols.success
   ) {
     // format lint output
+    let policyLines = ''
+    if (!result.passed) {
+      if (rulePolicyUrl)
```

Comment:
So your thinking is that the output will change on existing users, but only if they have set policyInfo/Url, which is probably as desired?

Reply:
This is intended. There is a bit of assumption behind this logic.
Customers using `symbol_formatter` _probably_ doesn't include policyUrl or policyInfo in their ruleset since they aren't displayed in the CLI output, so adding them here based on separate append logic should work well.